lib/tag-list.jsx

```diff
@@ -53,22 +53,31 @@ export class TagList extends Component {
     }
   };
 
+  renderEditTagsButton() {
+    if (this.props.tags.length) {
+      return (
+        <button
+          className="tag-list-edit-toggle button button-borderless"
+          tabIndex="0"
+          onClick={this.props.onEditTags}
+        >
+          {this.props.editingTags ? 'Done' : 'Edit'}
+        </button>
+      );
+    }
+    return null;
+  }
```

Comment:
I hate to be the 'semicolon person', but could you please add one here ;)

Reply:
no need to @roundhill - that's what `prettier` is for :wink: